src/modules/project/pages/projectView/views/projectMainBody/sections/Goals.tsx

```diff
@@ -82,7 +51,7 @@ export const Goals = () => {
       return completedGoals.map((goal: ProjectGoal) => {
         if (goal) {
           return (
-            <GoalCompleted key={goal.id} goal={goal} isEditing={editMode} onOpenGoalModal={handleEditGoalModalOpen} />
+            <GoalCompleted key={uuidv4()} goal={goal} isEditing={editMode} onOpenGoalModal={handleEditGoalModalOpen} />
```

Comment:
why did goalID did not work for these?

Reply:
oops, I was trying to solve another issue and I left this here. Nice catch!